entity-framework/core/providers/cosmos/index.md

```diff
@@ -70,374 +73,10 @@ The Azure Cosmos DB provider for EF Core has multiple overloads of the [UseCosmo
 | Account endpoint and token | `UseCosmos<DbContext>(accountEndpoint, tokenCredential, databaseName)` | [Resource tokens](/azure/cosmos-db/secure-access-to-data#primary-keys)                    |
```

Comment:
In the `IMPORTANT` above explicitly call out that RBAC is the recommended mechanism

Reply:
I'm actually not sure which one is RBAC in our table! Our table looks like [the one in the Cosmos docs](https://learn.microsoft.com/en-us/azure/cosmos-db/secure-access-to-data?tabs=using-primary-key), but doesn't exactly correspond...

May do a separate RBAC/auth/whatever PR to clean this stuff up?